src/advantest/smt8/origen/test_methods/Functional_test.java

```diff
@@ -109,13 +110,26 @@ public Functional_test overlaySubroutinePerSite(
     return this;
   }
 
+  //overload for full path pattern (from PGM_SOC_DATA) Boolean not used at the moment
+  public Functional_test overlaySubroutinePerSite(String subroutinePattern,
+          MultiSiteLong decData, int size, Boolean FullPathPattern ) {
+      Origen.overlaySubroutinePerSite(subroutinePattern,"NVM_DIN_PIN", decData, size);
+      return this;
+  }
+
   public Functional_test overlaySubroutineAllSites(
       String subroutinePattern, long decData, int size) {
     Origen.overlaySubroutineAllSites(
         "C402TNVMTester.patterns." + subroutinePattern, "NVM_DIN_PIN", decData, size);
     return this;
   }
 
+  //overload for full path pattern (from PGM_SOC_DATA) Boolean not used at the moment
+  public Functional_test overlaySubroutineAllSites(String subroutinePattern, long decData,int size, Boolean FullPathPattern) {
+      Origen.overlaySubroutineAllSites(subroutinePattern, "NVM_DIN_PIN", decData, size);
```

Comment:
As above

Reply:
reference to project specific pin has been removed. It is now passed in as arg